config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,32 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.kafka.brokers[0]
+  kafka_security_protocol = first_kafka_server_config.dig('authtype')
```

Comment:
Observation/Optional: This would crash if first broker config isn't set. Use safe navigation `&.dig(...`.

Reply:
Agree, I changed all the accesses to this object to use safe_navigation. However, it is true that if not even the first broker info is there , there will be bigger problems to solve :)